base/testfiles-lttemplates/lttemplates002.lvt

```diff
@@ -73,6 +73,30 @@
   \UseInstance{obj}{inst6}
 }
 
+\TEST { Expanding~values }
```

Comment:
Test should cover the full range of expansion variants.

Reply:
I'll add `c`, but everything else is just standard `\cs_generate_variant:Nn` stuff, so I don't see we have to cover everything.

Reply:
That said, it's not hard to cover a few more, so I will extend.